model-server/build.gradle.kts

```diff
@@ -117,8 +117,19 @@ val cucumber = task("cucumber") {
     }
 }
 
+tasks.register<Copy>("copyApis") {
+    from("../api/")
+    include("*.yaml")
+    into(project.layout.projectDirectory.dir("src/main/resources/api"))
```

Comment:
This should go under the build directory to avoid accidentally committing the generated files. The srcSets need to be extended then with the new source root that is created there.